src/runtime/config/configuration-qemu-nvidia-gpu-snp.toml.in

```diff
@@ -553,8 +553,8 @@ kernel_modules=[]
 #debug_console_enabled = true
 
 # Agent connection dialing timeout value in seconds
-# (default: 90)
-dial_timeout = 90
+# (default: @DEFAULTTIMEOUT_NV@)
+dial_timeout = @DEFAULTTIMEOUT_NV@
```

Comment:
This is not the timeout responsible for having failing the GPU, its the `create_container_timeout`, `dial_timeout` has no effect on this.

Reply:
There are no long running operations in the PodSandboxCreate

Reply:
Changing the default_* will create a discrepancy with the podoverhead. Those two things need to go hand in hand.